src/test/java/nextstep/fp/LambdaTest.java

```diff
@@ -33,19 +34,34 @@ public void runThread() throws Exception {
 
     @Test
     public void sumAll() throws Exception {
-        int sum = Lambda.sumAll(numbers);
+        int sum = Lambda.sum(numbers, new Conditional() {
+            @Override
+            public boolean test(int number) {
+                return true;
+            }
+        });
```

Comment:
```suggestion
        int sum = Lambda.sum(numbers, number -> true);
```
익명클래스도 람다식으로 변환하면 가독성있는 코드가 되겠네요 😄